packages/core-input/core-input.js

```diff
@@ -72,6 +72,7 @@ function onKey (input, event) {
     else if (event.keyCode !== KEYS.ENTER) input.focus()
   }
 
+  if (!item && !list.hasAttribute('hidden') && event.keyCode === KEYS.ESC) event.preventDefault()
```

Comment:
Why check `!item`? :)

Reply:
I thought that I should ensure that item was false when the user pressed ESC, but seeing it as the variable is never set to anything when the user actually does press ESC this check is unnecessary. Nice catch! Will remove :).